src/include/OpenImageIO/typedesc.h

```diff
@@ -529,3 +529,34 @@ convert_type(TypeDesc srctype, const void* src,
 
 
 OIIO_NAMESPACE_END
+
+
+
+#if OIIO_USE_FMT
+// Supply a fmtlib compatible custom formatter for TypeDesc.
+FMT_BEGIN_NAMESPACE
+template <>
+struct formatter<OIIO::TypeDesc> {
+    // Parses format specification
+    // C++14: constexpr auto parse(format_parse_context& ctx) {
```

Comment:
Did you mean to add `contexpr` here in c++14 mode ?

Reply:
No, I just meant for the comment to remind myself that when C++14 is our minimum, I can switch to the more compact declaration. I barely noticed the constexpr part, the big difference was that C++11 needs the "-> decltype(...)" to go along with the "auto", but C++14 can deduce the return type.

Reply:
I'm assuming that `constexpr` is desirable for when you want to use the compile time format string parsing feature - but I guess we are not using this widely right now?

Reply:
Not using it at all for the moment.